packages/components/svgo.spec.ts

```diff
@@ -35,7 +40,7 @@ describe("removeRootSVGElement", () => {
   }
 
   it("Replaces the <svg> element with its children", () => {
-    const rootItem = {
+    const rootItem: RootItem = {
```

Comment:
Maybe revert this file since I'll deal with it when I move it to v1

Reply:
If I revert this, Vitest can't parse it hmmm

```  FAIL  svgo.spec.ts [ svgo.spec.ts ]
@kaizen/components:test: Error: require() of ES Module /Users/geoff.chong/Code/kaizen-design-system/packages/components/svgoUtils.js from /Users/geoff.chong/Code/kaizen-design-system/packages/components/svgo.spec.ts not supported.```

Reply:
Might need to skip this one and just wait for your fix